packages/story-editor/src/app/media/uploadQueue/index.ts

```diff
@@ -934,17 +991,17 @@ function useMediaUploadQueue() {
      * Checks whether the resource is not external and
      * not already uploading.
      *
-     * @param {import('@googleforcreators/media').Resource} resource Resource object.
-     * @return {boolean} Whether a given resource can be transcoded.
+     * @param resource Resource object.
+     * @return Whether a given resource can be transcoded.
      */
-    const canTranscodeResource = (resource) => {
-      const { isExternal, id, src } = resource || {};
+    const canTranscodeResource = (resource: Resource) => {
```

Comment:
The return type of this one is a bit add, as it is `boolean | ''`. If we just change `src && ` to `src !== '' && `, it'll be just `boolean`.

Reply:
Fixed